packages/ember-runtime/lib/ext/rsvp.js

```diff
@@ -1,85 +1,57 @@
-import Ember from 'ember-metal/core'; // Ember.testing, Ember.onerror
-import require, { has } from 'require';
-import { assert } from 'ember-metal/debug';
-import Logger from 'ember-console';
-
-import run from 'ember-metal/run_loop';
 import * as RSVP from 'rsvp';
+import run from 'ember-metal/run_loop';
+import { assert } from 'ember-metal/debug';
+import { dispatchError } from 'ember-metal/error_handler';
 
-var testModuleName = 'ember-testing/test';
-var Test;
-
-var asyncStart = function() {
-  if (Ember.Test && Ember.Test.adapter) {
-    Ember.Test.adapter.asyncStart();
-  }
-};
-
-var asyncEnd = function() {
-  if (Ember.Test && Ember.Test.adapter) {
-    Ember.Test.adapter.asyncEnd();
-  }
-};
-
-RSVP.configure('async', function(callback, promise) {
-  var async = !run.currentRunLoop;
+const backburner = run.backburner;
+run._addQueue('rsvpAfter', 'destroy');
```

Comment:
Should we use a private symbol for this? I'd hate for folks to try and schedule into our special queue...